scanomatic/io/pickler.py

```diff
@@ -20,25 +20,6 @@ def safe_load(path, return_string=False):
         return fh
 
 
-def unpickle_with_unpickler(unpickler, path, *args, **kwargs):
-    """Compatibility unpickler that handles problems
-
-    This is used for
-
-    1) Dealing with errors in line-endings in data sent between different OS
-    2) Dealing with SoM refactoring import errors
-
-    Args:
-        unpickler (func): A function that unpickles a string of data
-        path (str): Path to the data to be unpickled
-        *args: Optional extra arguments for the unpickler
-        **kwargs: Opional extra keyword arguments for the unpickler
-
-    Returns: Unpickled object
-    """
-    return unpickler(safe_load(path), *args, **kwargs)
```

Comment:
is unpickler used now?

Reply:
No.

Reply:
Or well, `unpickler` was an argument to `unpickle_with_unpickler`, but I assume you mean `unpickle`, which is still used, though only in two places.

Reply:
`unpickle` only does this:
```def unpickle(path: str):
    """Unpickles data safely
    Returns: Unpickled object
    """
    return pickle.load(safe_load(path))
```

Reply:
yea probably nicer to inline

Reply:
fixed (next push)